### PR TITLE
Add instructions for secrets in CLI

### PR DESCRIPTION
**Pull request by bryannho**
- Added `Defining environment variables` section in `cli.md` to document support for secrets. Instructs users on how to configure secrets using `.env` file and deploy the project. Also links to `env_vars.md` for instructions on how to access and verify secrets from their application
- Added a note in `env_vars.md` that directs users to `cli.md` if they want to configure environment variables there

Related PR: https://github.com/ploomber/cli/pull/33

----
📚 Documentation preview 📚: https://ploomber-doc--98.org.readthedocs.build/en/98/

### CODE REVIEW

**Comment by bryannho** [comment]
@edublancas added some documentation on secrets in `cli.md`. There was already sample code in `env_vars.md` that checks if a given env var is defined and returns a response, so I just linked to it. The docs build is giving errors when linking to specific sections in other pages, so I just linked to the other page as a whole. Let me know if it's clear enough

File: doc/user-guide/cli.md [hunk 1]
```diff
@@ -64,4 +64,29 @@ Then, execute:
 
 ```sh
 ploomber-cloud deploy
-```
\ No newline at end of file
+```
+
+## Defining environment variables
```
Comment (edublancas) [inline doc/user-guide/cli.md:69]: don't forget to mention that the `.env` file is not uploaded and that's replaced by an empty file at runtime for security reasons

File: doc/user-guide/cli.md [hunk 2]
```diff
@@ -64,4 +64,29 @@ Then, execute:
...
+
+If your project makes use of environment variables, you can define them in an `.env` file. Simply run:
```
[the remark this answers is not in the document]
Comment (bryannho) [reply doc/user-guide/cli.md]: [Made this change](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R72)

File: doc/user-guide/cli.md [hunk 3]
```diff
@@ -64,4 +64,29 @@ Then, execute:
...
+
+```sh
+touch .env
```
[the remark this answers is not in the document]
Comment (bryannho) [reply doc/user-guide/cli.md]: [Removed touch command](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R72-R74). Also mentioned that the `.env` file should be created in the main project directory.

File: doc/user-guide/cli.md [hunk 4]
```diff
@@ -64,4 +64,29 @@ Then, execute:
...
+```
+
+Then open `.env` in your code editor and enter your environment variables. It should look like this:
+
+```
+MY_ENV_VAR_1=value_1
+MY_ENV_VAR_2=value_2
+```
+
+Now make sure your project has been initialized, and deploy it:
```
[the remark this answers is not in the document]
Comment (bryannho) [reply doc/user-guide/cli.md]: [Added the link](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R81)

File: doc/user-guide/cli.md [hunk 5]
```diff
@@ -64,4 +64,29 @@ Then, execute:
...
+
+```sh
+ploomber-cloud deploy
+```
+
+The command-line interface will automatically read your environment variables and include them in the deployment.
```
[the remark this answers is not in the document]
Comment (bryannho) [reply doc/user-guide/cli.md]: Mentioned this [here](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R87-R88)

File: doc/user-guide/cli.md (hunk 1 above)
Comment (bryannho) [reply doc/user-guide/cli.md:70]: [Added a mention for this](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R87-R88).

**Review by edublancas — APPROVED** [review]

**Pull request merged**